cli/ts/commands/deployPoll.ts

```diff
@@ -43,38 +51,56 @@ export const deployPoll = async (
     logError("Please provide a VkRegistry contract address");
   }
 
-  const vkRegistry = vkRegistryAddress ? vkRegistryAddress : vkRegistryContractAddress;
+  const vkRegistry = vkRegistryAddress || vkRegistryContractAddress;
 
-  const _maciAddress = readContractAddress("MACI");
-  if (!_maciAddress && !maciAddress) {
+  const maciContractAddress = readContractAddress("MACI");
+  if (!maciContractAddress && !maciAddress) {
     logError("Please provide a MACI contract address");
   }
 
-  const _maci = maciAddress ? maciAddress : _maciAddress;
+  const maci = maciAddress || maciContractAddress;
```

Comment:
should it be `??` here instead of `||`? I think they have slight difference

Reply:
No, for all the cases `||` is more safe option.

Reply:
then for this:
https://github.com/privacy-scaling-explorations/maci/blob/941d694835c4dab16448a7748985e2d55ddf192f/cli/ts/commands/genLocalState.ts#L81
we use `??` here is because of the `blockNumber` might be 0?
just wanna make sure for future coding styles~

Reply:
Actually, it depends on the case. For current one, I don't want to use empty string as address. For block number, if block number is zero - it won't fall further.